src/shared/queries.ts

```diff
@@ -102,8 +102,8 @@ export const selectAccountByEmail = gql`
 `
 
 export const selectAccountByTicket = gql`
-  query($ticket: uuid!) {
-    auth_accounts(where: { ticket: { _eq: $ticket } }) {
+  query($ticket: uuid!, $now: timestamptz!) {
+    auth_accounts(where: { _and: [{ ticket: { _eq: $ticket } }, { ticket_expires_at: { _gt: $now } }] }) {
```

Comment:
was this selectAccountByTicket used somewhere where `ticket_expires_at` was not important? I'm just thinking so we don't break anything else.

Reply:
Nope - we would have caught that on the tests.